extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -128,12 +136,94 @@ function PanelMeasurementTableTracking({ servicesManager, extensionManager }) {
     DICOMSR.downloadReport(trackedMeasurements, dataSource);
   }
 
+  const jumpToImage = id => {
+    onMeasurementItemClickHandler(id);
+
+    const measurement = measurements.find(m => m.id === id);
```

Comment:
Instead of caching these here, use the `id` to get a specific measurement from the `MeasurementService`? Prevent uneccessary render when we call `setMeasurements` and then we don't have to duplicate state

Reply:
Updated